packages/desktop-client/src/components/reports/graphs/sankey-spreadsheet.tsx

```diff
@@ -0,0 +1,191 @@
+import q, { runQuery } from 'loot-core/src/client/query-helpers';
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏thanks for creating this file with TS